draft-ietf-tls-tls13.md

```diff
@@ -3609,26 +3620,37 @@ operation.
    PSK ->  HKDF-Extract
                  |
                  v
-           Early Secret ---> Derive-Secret(., "early traffic secret",
+           Early Secret ---> Derive-Secret(., "client early traffic secret",
                  |                         ClientHello)
-                 |                         = early_traffic_secret
+                 |                         = client_early_traffic_secret
                  v
 (EC)DHE -> HKDF-Extract
                  |
                  v
               Handshake
-               Secret -----> Derive-Secret(., "handshake traffic secret",
+               Secret
```

Comment:
Unbelievably nitpicky formatting nit: I think we can fit "Handshake Secret" comfortably on one line.